Amazon.IonDotnet.Tests/Common/ReaderTestCommon.cs

```diff
@@ -202,6 +216,20 @@ public static void ReadTypeAnnotationSymbols_SingleField(IIonReader reader)
             }
         }
 
+        public static void ReadTypeAnnotationSymbols_ZeroSymbol(IIonReader reader)
+        {
+            // an int with zero symbol annotation
+            // $0::18
+            reader.MoveNext();
+
+            Assert.AreEqual(IonType.Int, reader.CurrentType);
+            Assert.AreEqual(18, reader.IntValue());
+
+            Assert.AreEqual(1, reader.GetTypeAnnotationSymbols().Count());
+
+            Assert.IsTrue(reader.GetTypeAnnotationSymbols().Any(a => a.Text == null));
```

Comment:
Also need to check that the import location is null.

Reply:
Did you mean SID? When I assert `a.ImportLocation == null`, it fails 

The import location is: `{ImportLocation::{importName:, id:0}}`

Reply:
Looks like a bug. SID 0 should not have an import location.

The fix can be in a separate CR if you want. Please create an issue.

Reply:
Created issue: https://github.com/amzn/ion-dotnet/issues/89

Could you set milestone?